iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/vector/HiveBatchIterator.java

```diff
@@ -123,6 +124,13 @@ private void advance() {
               lcv.isRepeating = false;
               System.arraycopy(value, 0, lcv.vector, 0, batch.size);
               break;
+            // Do not fill anything while evaluating deletes.
```

Comment:
why do we need this comment? i don't see any checks for deletes here

Reply:
Removed.